perseo_core-entrypoint.sh

```diff
@@ -1,13 +0,0 @@
-#!/bin/bash
```

Comment:
The current approach based in this entry point is tough to set URL as paramter. For instance, when used in docker-compose.yml we typically use in the perseo-core image something like this:

```
  command: -perseo_fe_url http://127.0.0.1:9090
```

I understand that the changes in this PR would need adaptation in other places using perseo-core docker (e.g. using `environment` sention in docker-compose.yml instead of `command`), involing a risk in backward compatibility with deployment systems based in this approach.

Would it be possible to implemente env vars based configuration but a the same time preserve the current mechanism based in entrypoint? I mean, if both are used env vars will take preference. But if env vars are not used the URL configured via entrypoint will be used.

Reply:
OK. I understand the problems that could result in eliminating the entrypoint, I think the solution you propose is fine. I will add the changes to the branch.